reviewpad.yml

```diff
@@ -61,7 +61,7 @@ workflows:
     if:
       - rule: $hasFilePattern("public/data/**")
     then:
-      - '$addLabel("chore: profile-adition")'
+      - '$addLabel("chore: profile-addition")'
```

Comment:
It'd be best if we could use `✍ chore: profile addition`; not sure if that's possible, though.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
I have now added the icon too.